plugins/chat/app/jobs/service_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ServiceJob < ::Jobs::Base
+  include Chat::WithServiceHelper
+
+  def run_service(service, dependencies)
```

Comment:
We just need to overload this method as we don’t have any `params` or `guardian` inside a job.

Reply:
If we did want to pass guardian, it can just go in dependencies right? e.g. `with_service(Blah, args.merge(guardian: Guardian.new(User.find(args[:user_id])))`

Reply:
It's an interesting question, because if you have a service used in a controller and in a job, one might have the guardian set the other would have to fetch it. And then you have to handle some logic, if no guardian... get guardian.

So I would say setting it from the job is probably a simpler.